cornell-dti/course-plan
# [EZ] Update CS Tech Electives & INFO Electives to be 3000+, not 3000 only

### Summary 

Update the CS major technical electives and CS electives checker to include courses that have a course number 3000+ or 4000+ respectively, and not just starting with a 3 or 4. Furthermore, update the INFO Electives requirement to use this new way of checking these courses to be more lenient

- [x] Updated CS technical and CS electives checkers
- [x] Updated Info Electives checker
- [x] Ran req gen

![image](https://user-images.githubusercontent.com/25535093/111881310-415a6c00-8986-11eb-8e36-50c5ab11e6b4.png)

Remaining TODOs:

- [ ] This fixes the issue of courses 4000 level or above not being able to be applied properly to technical requirements, but does not fix the more complicated issue of 2000 or below courses that can be selected but should not be. This will be covered in the next sprint.

### Test Plan 

1. Confirm not only 3000 but 4000+ courses can be added to the technical req self check and INFO elective (if INFO classes), and 5000+ CS courses can be added to the CS elective. Ensure that the course shows up under the requirement name and the counter increments for the self check ones.
2. Do the same for courses under 3000 for the technical self check, and confirm the counter does not increment.
3. Confirm that courses that should be exceptions for CS electives and INFO electives are actually exceptions and either do not count when added to, or are not allowed to be added to. Previous bugs in the code seem to have missed some exceptions.
